css/style.css

```diff
@@ -38,4 +38,26 @@
     left: 0;
     width: 100%;
     height: 100%;
-}
\ No newline at end of file
+}
+
+.nested-view{
+    width:100%;
+    height:500px;
+    background-color: #cccccc;
+    margin-top: 10px;
+}
+
+.close-button{
+    background: transparent none repeat scroll 0% 0%;
+    border: 0px none;
+    padding: 0px;
+    outline: medium none;
+}
+
+.h-center{
```

Comment:
maybe it is better to call centered-text or something like this (if it is applicable not only for h elements)